pkg/standalone/standalone.go

```diff
@@ -85,8 +86,11 @@ const (
 	healthPort = 58080
 	metricPort = 59090
 
-	schedulerHealthPort = 58181
-	schedulerMetricPort = 59191
+	schedulerHealthPort = 58081
+	schedulerMetricPort = 59091
+	schedulerEtcdPort   = 52379
+
+	daprVersionsWithScheduler = ">= 1.13.x"
```

Comment:
I had to change this to `>= 1.12.x` to test. It requires a valid Dapr release in GitHub and an existing dapr image in the container registry. So, I cannot test with 1.14.x.